// python -m grpc_tools.protoc -I./ --python_out=. --pyi_out=. --grpc_python_out=. ./proto.proto

syntax = "proto3";

option java_package = "ex.grpc";
option objc_class_prefix = "HSW";

package proto;

service LlamaCallback {
  rpc llamaAsk (LlamaRequest) returns (stream LlamaReply) {}
}

message LlamaRequest {
  string prompt = 1;
}

message LlamaReply {
  string answer = 1;
}
